### ISSUE TITLE
Add page titles to each page

### ISSUE THREAD

**Issue by gotonode** [UniversityOfHelsinkiCS/labtool#940]
This is an accessibility issue. Each page should have their own title, preferably unique.

For an example, the main page could be titled "Labtool", and the import page could be titled "Import Courses - Labtool" or "Labtool - Import Courses". There are different schools of thought regarding which is more appropriate. Personally, I prefer the most specific part "Import Courses" to precede the general part that is on every page, "Labtool".

I'll close #845 since that is way too broad. Changes to accessibility should be done while working on any specific feature, and it's a constant process.

**Comment by hisahi** [comment]
Done in #990
